Automattic/wp-calypso
# P2: Better error message when trying to delete a P2 hub with spaces

#### Changes proposed in this Pull Request

A recent update to the API will refuse to delete a P2 hub if it has spaces. See `D63635-code`.

The error message that is displayed directly from the 403 from the API does not hang out on the screen for very long, so this is an error message that is there until the user dismisses it. It has an error explication and a link for the user to go fix the problem.

#### Testing instructions

* On a P2 hub, go to `http://calypso.localhost:3000/settings/delete-site/[your-site]`.
* Try deleting the site by clicking "Delete Site" - you should get an error in the upper right corner that stays and only goes away when dismissed.



##### Before
<img width="639" alt="before" src="https://user-images.githubusercontent.com/193283/124300086-75852180-db5e-11eb-91ba-b7c44c2b05f4.png">

##### After
<img width="832" alt="after" src="https://user-images.githubusercontent.com/193283/124300506-f0e6d300-db5e-11eb-8f94-7041d450b277.png">
